api/db-cdr/changelog/db.changelog-11.xml

```diff
@@ -0,0 +1,167 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.0.xsd
+        http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd">
+
+  <changeSet author="biopete" id="changelog-11.0"  runAlways="true" >
+    <validCheckSum>ANY</validCheckSum>
+    <preConditions onFail="MARK_RAN">
+      <tableExists tableName="ACHILLES_analysis" schemaName="cdr"></tableExists>
+    </preConditions>
+    <dropTable tableName="ACHILLES_analysis"/>
+  </changeSet>
+
+  <changeSet author="biopete" id="changelog-11.1" runAlways="true" >
+    <validCheckSum>ANY</validCheckSum>
+    <preConditions>
+      <not>
+        <tableExists tableName="ACHILLES_analysis" schemaName="cdr"></tableExists>
+      </not>
+    </preConditions>
+
+    <createTable tableName="ACHILLES_analysis">
+      <column name="analysis_id" type="BIGINT">
+        <constraints primaryKey="true" nullable="false"/>
+      </column>
+      <column name="analysis_name" type="varchar(300)">
+        <constraints nullable="false"/>
+      </column>
+      <column name="stratum_1_name" type="VARCHAR(255)">
+        <constraints nullable="true"/>
+      </column>
+      <column name="stratum_2_name" type="VARCHAR(255)">
+        <constraints nullable="true"/>
+      </column>
+      <column name="stratum_3_name" type="VARCHAR(255)">
+        <constraints nullable="true"/>
+      </column>
+      <column name="stratum_4_name" type="VARCHAR(255)">
+        <constraints nullable="true"/>
+      </column>
+      <column name="stratum_5_name" type="VARCHAR(255)">
+        <constraints nullable="true"/>
+      </column>
+      <column name="chartType" type="VARCHAR(20)">
+        <constraints nullable="true"/>
+      </column>
+      <column name="dataType" type="VARCHAR(20)">
+        <constraints nullable="true"/>
+      </column>
+    </createTable>
+
+  </changeSet>
+
+
+  <changeSet author="biopete" id="changelog-11.2" runAlways="true">
+    <validCheckSum>ANY</validCheckSum>
+    <loadData tableName="ACHILLES_analysis" file="csv/ACHILLES_analysis.csv" encoding="UTF-8" quotchar='"'>
```

Comment:
See #workbench-eng thread about loadData. We're probably going to want to have this tagged in such a way that it can run separately from the schema changes. It may also make sense to put it in another directory.

Reply:
If you want to let @freemabd1  worry about this part, that's fine with me. :)

Reply:
Also, can we name the csv files in lower case as well? ex: achilles_analysis.csv. I can take care of splitting the schema and data.

Reply:
same on lower case. Can't really do it unless you want the file different from table name.

That is fine if you take care of it Brian. For development purposes , I want it to be loaded every time since i will probably  be changing it